### PR TITLE
Add oncokb_api_bearer_token in command line and refactor main()

### PR DESCRIPTION
**Pull request by victoria34**
Fixed 
https://github.com/oncokb/oncokb/issues/1763
https://github.com/oncokb/oncokb/issues/1736

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ Example input files are under [data](data). An example script is here: [example.
 
 We recommend processing MAF files by [vcf2maf](https://github.com/mskcc/vcf2maf/) with [MSK override isoforms](https://github.com/mskcc/vcf2maf/blob/master/data/isoform_overrides_at_mskcc) before using the `MafAnnotator` here.
 
+## OncoKB API
```
Comment (zhx828) [inline README.md]: Let's move this section to the top in the first release to emphasis the importance and also add a section header `UPDATE: API token required`

File: CnaAnnotator.py [hunk 2]
```diff
@@ -2,71 +2,54 @@
...
+        print '     3) default tumor type (-t)'
+        print '  Default OncoKB base url is http://oncokb.org'
+        sys.exit()
+    if argv.input_file == '' or argv.output_file == '' or argv.oncokb_api_bearer_token == '':
         print 'for help: python CnaAnnotator.py -h'
         sys.exit(2)
-
-    for opt, arg in opts:
-        if opt == '-h':
-            print 'CnaAnnotator.py -i <input cNA file> -o <output MAF file> [-p previous results] [-c <input clinical file>] [-s sample list filter] [-t <default tumor type>] [-u base-url]'
```
Comment (zhx828) [inline CnaAnnotator.py]: Good catch

**Comment by zhx828** [comment]
@victoria34 this pr will not be merged before the oncokb public release.

File: README.md (hunk 1 above)
Comment (victoria34) [reply README.md]: @zhx828 Moved  UPDATE API section to top.
Screen Shot 2019-10-27 at 5 35 43 PM

File: README.md [hunk 3]
```diff
@@ -12,6 +12,13 @@ Example input files are under [data](data). An example script is here: [example.
 
 We recommend processing MAF files by [vcf2maf](https://github.com/mskcc/vcf2maf/) with [MSK override isoforms](https://github.com/mskcc/vcf2maf/blob/master/data/isoform_overrides_at_mskcc) before using the `MafAnnotator` here.
 
+## UPDATE: API token required
+#### OncoKB API
+When you run `MafAnnotator.py`, `FusionAnnotator.py` and `CnaAnnotator.py`, you need a token before access the OncoKB data via its web API. Please go to [OncoKB Data Access](https://oncokb.org/dataAccess) to learn how to get a OncoKB API token and specify your token  in the command line after you have that.
```
[the remark this answers is not in the document]
Comment (victoria34) [reply README.md]: Updated

File: example.sh [hunk 4]
```diff
@@ -8,8 +8,9 @@ OCNA="data/example_cna.oncokb.txt"
 IC="data/example_clinical.txt"
 OC="data/example_clinical.oncokb.txt"
 OCPDF="data/example_clinical.oncokb.pdf"
-python MafAnnotator.py -i $IMAF -o $OMAF -c $IC
-python FusionAnnotator.py -i $IF -o $OF -c $IC
-python CnaAnnotator.py -i $ICNA -o $OCNA -c $IC
+OT="" #OncoKB API Token
```
[the remark this answers is not in the document]
Comment (victoria34) [reply example.sh]: Renamed

**Pull request merged**